learning/basics/task1/PW-input_print.py

```diff
@@ -0,0 +1,9 @@
+name = str(input("Hi there. What's your name? "))
```

Comment:
There is no need to type str() command beacuse result of input is always a string